app/components/Mods/ModsToolbar.tsx

```diff
@@ -18,8 +19,9 @@ const useStyles = makeStyles((theme) => ({
 
 const ModsToolbar: React.FunctionComponent = () => {
   const styles = useStyles();
+  const alpha = useRecoilValue(alphaState);
```

Comment:
it's convention to prefix flags to make code more readable (`is`, `has`, etc). In this case it should be `isAlpha`. Also, don't be afraid of long variable names. The more descriptive the better. Like `isAlphaModsEnabled` or whatever you feel drescribes more precisely what this flag does, to for instance distinguish it from. Especially when you already have a `mod.isAlpha`, so you want to always be able to very clearly tell what you're looking at.